doc/docs/json-schema.md

```diff
@@ -65,6 +66,7 @@ import sttp.tapir.generic.auto._
   val schemaStr: String = Printer.spaces2.print(schemaAsJson.deepDropNullValues)
 ```
 
+The title annotation of the object will be by default the name of the case class. You can customize it with `@title` annotation.
```

Comment:
hm do we want this to be the default? Is this somehow mandated by json schema, that the title is present? Maybe this should be an option of the serialiser, similar to `markOptionsAsNullable: Boolean`?

Reply:
Sure, what do you propose as the name of the option?

Reply:
`addTitleToDefs`? :) I guess that's where the title is added.

Still an open question what should be the default. Maybe there are some examples of json schemas out there, for similar data structures, do they have the `title` set?

Reply:
Ok, let's set it by default to `true` when generating json-schema